agent/bootstrap.sh

```diff
@@ -66,6 +66,15 @@ CFLAGS='-g -O0 -ftrapv' meson build \
       -Dnobody-user=nfsnobody \
       -Dnobody-group=nfsnobody
 ninja-build -C build
+
+# Let's check if the new systemd at least boots before installing it
+# As the CentOS' systemd-nspawn version is too old, we have to use QEMU
+INITRD_PATH="/boot/initramfs-$(uname -r).img"
+KERNEL_PATH="/boot/vmlinuz-$(uname -r)"
+[ ! -f /usr/bin/qemu-kvm ] && ln -s /usr/libexec/qemu-kvm /usr/bin/qemu-kvm
+make -C test/TEST-01-BASIC clean setup run clean-again TEST_NO_NSPAWN=1 INITRD=$INITRD_PATH KERNEL_BIN=$KERNEL_PATH KERNEL_APPEND="debug"
```

Comment:
It would certainly have helped in https://github.com/systemd/systemd/pull/11178 and looks great overall. I'd turn SELinux off with `setenforce 0` before running the test, though.

Reply:
Sorry. I apparently missed `TEST_NO_NSPAWN=1`. Running the test with QEMU should be fine with SELinux enforced.